src/style.css

```
@@ -0,0 +1,113 @@
+html,
+body,
+form,
+img,
+div {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  font-style: normal;
+  font-weight: 400;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.lg-search-bar__form,
+.sm-search-bar__form {
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+  margin-left: 8px;
+}
+
+.lg-search-bar__icon {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  position: absolute;
+  left: 0;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+  margin: 25px 17px 26px 26px;
+}
+
+.lg-search-bar__input,
+.sm-search-bar__input {
+  position: relative;
+  z-index: 0;
+
```

Input will take full width as parent form
```suggestion
  width: 100%;
```